st2common/st2common/middleware/auth.py

```diff
@@ -77,20 +76,8 @@ def _validate_token(self, env):
         token_in_headers = env.get('HTTP_X_AUTH_TOKEN', None)
```

Comment:
This is an environment variable name set by wsgi - as above, I didn't touch or change key name.